temel_bilgiler/javascript_temelleri/fundamentals-1.md

```diff
@@ -0,0 +1,118 @@
+
+1. [Bu W3Schools dersi](https://www.w3schools.com/js/js_arithmetic.asp) ve ardından [bu ders](https://www.w3schools.com/js/js_numbers.asp), JavaScript'te sayılarla neler yapabileceğinize iyi bir giriş yapar.
+2. [Bu MDN makalesi](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math), temel matematiği JavaScript'te nasıl uygulayacağınızı öğretirken farklı bir bakış açısından aynı bilgileri kapsar. Sayılarla yapabileceğiniz çok daha fazlası var, ancak şimdilik bunların hepsi ihtiyacınız olanlar.
+3. JavaScript'te sayılarla (diğer şeyler arasında!) ne yapabileceğinizi oldukça iyi bir şekilde gösteren [bu makaleyi](http://javascript.info/operators) okuyun (ve kodla birlikte çalışın). Sayılarla neler yapabileceğinize dair oldukça iyi bir fikir edineceksiniz.
+
+
+### Görev
+
+<div class="lesson-content__panel" markdown="1">
+Aşağıdaki egzersizleri deneyin (ve `console.log()` kullanmayı unutmayın!):
+
+1. İki sayıyı toplayın! (html dosyanızda `console.log(23 + 97)` yazın)
+2. Farklı 6 sayı dizisini bir araya getirin.
+3. Aşağıdaki denklemi çözümleyin: `(4 + 6 + 9) / 77`
+    * Cevap yaklaşık `0.24675` olmalıdır.
+4. Hadi değişkenleri kullanalım!
+    * Script etiketinin üstüne şu ifadeyi yazın: `let a = 10`
+    * Konsola `console.log(a)` yazarsanız `10` yazısı gelmelidir.
+    * Konsolda şunu deneyin: `9 * a`
+    * ve bunu: `let b = 7 * a` ( `undefined` \* değerini döndürür) ve sonra `console.log(b)` yazın.
+5. Şimdiye kadar bu işi halletmeye başladınız... şunu deneyin:
+    * Sabit bir değişken olan `max`ı `57` değeriyle belirtin.
+    * Başka bir değişken olan `actual`ı `max - 13` olarak ayarlayın.
+    * Başka bir değişken olan `percentage`i `actual / max` olarak ayarlayın.
+    * Eğer konsola `percentage` yazarsanız ve enter tuşuna basarsanız `0.7719` gibi bir değer göreceksiniz.
+6. Script etiketinizde çeşitli şeylerle oynamaya birkaç dakika daha ayırın. Sonunda, bu sayıların ve şeylerin gerçekten web sayfasında görünmesini nasıl yapacağımızı öğreneceğiz, ancak tüm bu mantık aynı kalacak, bu nedenle devam etmeden önce rahat olduğunuzdan emin olun.
+
+_* Javascript kodunu konsolda çalıştırarak fark ettiğiniz gibi, konsol, yürüttüğü kodun sonucunu yazdırır (döndürme deyimi denir). Bu konuda daha fazla bilgiyi sonraki derslerde öğreneceksiniz, ancak şimdilik bir atama ile deklarasyon (örneğin `let b = 7 * a`) `undefined` değerini döndürür, bu nedenle bir değişkene bir değer atayamaz ve aynı satırda değerini okuyamazsınız._
+</div>
+
+### Ek Kaynaklar
+
+Bu bölüm, diğer içeriklere yardımcı olan faydalı bağlantılar içerir. Gereksiz değildir, bu yüzden ek olarak düşünün.
+
+* `var` ve `let` arasındaki kesin farklar [javascript.info](https://javascript.info/var) adresinde açıklanmaktadır.
+
+### Bilgi Kontrolü
+
+Bu bölüm, bu derste kendi anlayışınızı kontrol etmeniz için sorular içerir. Bir soruyu yanıtlayamıyorsanız, tıklayın ve bağlantı verilen materyali gözden geçirin.
```

Comment:
Bu bölüm, bu dersi anlayıp anlamadığınızı kendi başınıza kontrol etmeniz için sorular içermektedir. Bir soruyu yanıtlamakta sorun yaşıyorsanız, soruya tıklayın ve bağlantının verdiği materyali inceleyin.